bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/MailingListBridgeBot.java

```diff
@@ -203,6 +203,7 @@ public List<WorkItem> getPeriodicItems() {
 
         List<PullRequest> prs = poller.updatedPullRequests();
         prs.stream()
+                .filter(pr -> !pr.isDraft())
```

Comment:
Are you sure this is the right place to do this? The intention of this fix is to pause notifications for Draft PRs until it becomes "rfr" again, just like it currently does for PRs initially in Draft that become "rfr" for the first time. Will it do this or will it just lose the notifications permanently?